### ISSUE TITLE
All node checks fail on a kubernetes 1.9 cluster

### ISSUE THREAD

**Issue by ttousai** [aquasecurity/kube-bench#129]
This cluster was deployed with kops.

```
kubectl run --rm -i -t kube-bench-node --image=aquasec/kube-bench:latest --restart=Never --overrides="{ \"apiVersion\": \"v1\", \"spec\": { \"hostPID\": true } }" -- node --version 1.8

....
== Summary ==
0 checks PASS
21 checks FAIL
2 checks WARN
```

**Comment by lizrice** [comment]
@ttousai assigning to you in the hope you're investigating already - let me know if you need help

**Comment by sauryadas** [comment]
I see that kube-bench supports 1.6 to 1.8 only . Is support planned for 1.9, 1.10 and 1.11?

kube-bench supports the tests for multiple versions of Kubernetes (1.6, 1.7 and 1.8) as defined in the CIS Benchmarks 1.0.0, 1.1.0 and 1.2.0 respectively.

**Comment by ttousai** [comment]
@sauryadas kube-bench currently supports only the official CIS benchmarks.

We will implement support for the more recent kubernetes versions as soon as CIS releases official benchmarks for them.

You can also run the tests for version 1.8 against more recent versions of kubernetes, but this could lead to some unexpected results and unchecked issues.

**Comment by phosphre** [comment]
Hello it appears [1.11 is covered](https://www.cisecurity.org/benchmark/kubernetes/).

**Comment by ttousai** [comment]
@phosphre CIS has released benchmarks for k8s 1.11 and we have a PR to add support https://github.com/aquasecurity/kube-bench/pull/163.

**Comment by phosphre** [comment]
Great work!
